### ISSUE TITLE
dump-autoload step needed

### ISSUE THREAD

**Issue by judgej** [webpatser/laravel-countries#37]
Before this step in the installation:

```
$ php artisan migrate --seed
```

I needed to do this:

```
composer.phar dump-autoload
```

Without this, I get the following error:

``` text
-bash-4.1$ php artisan migrate --seed
Nothing to migrate.
  [ReflectionException]
  Class CountriesSeeder does not exist
```

This is a common issue with Laravel when doing:

```
php artisan vendor:publish
```

**Comment by webpatser** [comment]
This step is now available in the master branch. See #78

**Comment by judgej** [comment]
I have a feeling this is fixed in Laravel 5.5, which does a lot more auto-discovery than previous versions, and puts more scripts into namespaces then any previous version. This `dump-autoload` can't do any harm though :-)
